app/controllers/user_games_controller.rb

```diff
@@ -5,10 +5,10 @@ def create
     if !current_user 
       flash[:notice] = "You must sign in to add games to Saved Games"
     elsif UserGame.user_game_exists?(game.id, current_user.id)
-      flash[:notice] = "You have already added #{game.title} to Saved Games"
+      flash[:notice] = "You have already added #{game.name} to Saved Games"
```

Comment:
`name` is returned by the API, might as well use that instead of `title` for consistency